modules/end-user-guide/pages/navigating-openshift-web-console-from-che.adoc

```diff
@@ -0,0 +1,24 @@
+:navtitle: Navigating OpenShift web console from {prod-short}
+:keywords: overview, OpenShift, web console
+:page-aliases: .:accessing-openshift-web-console-from-che, overview:accessing-openshift-web-console-from-che, accessing-openshift-web-console-from-che
+
+:parent-context-of-accessing-che-from-openshift-developer-perspective: {context}
+
+[id="navigating-openshift-web-console-from-{prod-id-short_{context}"]
+= Navigating OpenShift web console from {prod-short}
+
+This section describes how to access OpenShift web console from {prod-short}.
+
+.Prerequisites
+
+* The {prod-short} Operator is available in OpenShift 4.
+
+.Procedure
+
+. Open the {prod-short} dashboard and click the three-by-three matrix icon in the upper right corner of the main screen.
```

Comment:
```suggestion
1. Open the {prod-short} dashboard and click the three-by-three matrix icon in the upper right corner of the main screen.
```

Reply:
@swatikarmarkar The syntax `. something` is correct.